utils/build-script-impl

```diff
@@ -152,6 +152,7 @@ KNOWN_SETTINGS=(
     skip-test-android-host      ""               "set to skip testing the host parts of the Android toolchain"
     validation-test             "0"              "set to run the validation test suite"
     long-test                   "0"              "set to run the long test suite"
+    test-paths                  ""               "run tests located in specific directories and/or files"
```

Comment:
I'm hesitant to add any new options to `build-script-impl`, instead favoring additions to `build-script` if at all possible. However, since the rest of the testing infrastructure is in `build-script-impl` then it makes some sense to put this new logic here.